server/routes/approvingLicences/handlers/approvalView.ts

```diff
@@ -10,7 +11,9 @@ export default class ApprovalViewRoutes {
 
     // Check whether this licence is still in a SUBMITTED state - back button pressed - avoid re-approval
     if (licence?.statusCode === LicenceStatus.SUBMITTED) {
-      res.render('pages/approve/view')
+      const expandedLicenceConditions = expandAdditionalConditions(licence.additionalLicenceConditions)
+      const expandedPssConditions = expandAdditionalConditions(licence.additionalPssConditions)
+      res.render('pages/approve/view', { expandedLicenceConditions, expandedPssConditions })
     } else {
       res.redirect(`/licence/approve/cases`)
     }
```

Comment:
Expands the conditions for in-line text and data, and passes through to the views.